lighthouse-core/computed/resource-summary.js

```diff
@@ -51,6 +51,12 @@ class ResourceSummary {
 
     for (const record of networkRecords) {
       const type = this.determineResourceType(record);
+      if (type === 'other' && new URL(record.url).pathname === '/favicon.ico') {
```

Comment:
nit: we had to chase down all the places we did `new URL` on *every* network record once because it was hella slow in some cases, maybe `.endsWith('/favicon.ico)` plus `other` is good enough?